File: app_admin/lib/models/service.dart
class Service {
  const Service({
    required this.id,
    required this.name,
    required this.durationMinutes,
    required this.isActive,
    this.priceCents,
  });

  final String id;
  final String name;
  final int durationMinutes;
  final bool isActive;
  final int? priceCents;

  factory Service.fromJson(Map<String, dynamic> json) {
    return Service(
      id: json['id'] as String,
      name: json['name'] as String,
      durationMinutes: json['duration_minutes'] as int,
      isActive: json['is_active'] as bool,
      priceCents: json['price_cents'] as int?,
    );
  }
}

File: backend/app/models/service.py
import uuid

from sqlalchemy import Boolean, ForeignKey, Integer, String
from sqlalchemy.dialects.postgresql import UUID
from sqlalchemy.orm import Mapped, mapped_column

from app.models.base import Base, TimestampMixin, UUIDPKMixin


class Service(UUIDPKMixin, TimestampMixin, Base):
    __tablename__ = "services"

    tenant_id: Mapped[uuid.UUID] = mapped_column(
        UUID(as_uuid=True), ForeignKey("tenants.id", ondelete="CASCADE"), nullable=False
    )
    name: Mapped[str] = mapped_column(String, nullable=False)
    duration_minutes: Mapped[int] = mapped_column(Integer, nullable=False)
    price_cents: Mapped[int | None] = mapped_column(Integer, nullable=True)
    is_active: Mapped[bool] = mapped_column(Boolean, nullable=False, default=True)
